sdk/tests/conformance/textures/misc/texture-video-transparent.html

```diff
@@ -123,7 +123,7 @@
             testFailed("right is blue, but: " + redBlueError);
         }
     } else {
-        testFailed("neither left is green nor right is blue")
+        testFailed("neither left is green nor right is blue \n" + greenRedError + "\n" + redBlueError);
```

Comment:
It seems to me that if the only error is that the left side isn't green, that this detail log won't show any more information.